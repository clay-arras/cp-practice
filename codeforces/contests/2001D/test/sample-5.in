1
8
5 5 1 6 25 23 1 23
