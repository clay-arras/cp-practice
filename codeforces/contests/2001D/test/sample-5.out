5
5 1 6 25 23
